csun-metalab/nomi
# NAM-541 -- UI: Have check for nonexistent last names on front end for gallery cards

# Description
Ensure validation happens for null & undefined names.
  
# Testing
1. Visit student list, gallery cards, and flash cards and make sure no first or last names have undefined.
2. Ensure there is no extra period when there is no last-name abbreviation.
3. Visit student profile with no last name and ensure there is no undefined last-name.
